src/MetalsFeatures.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  StaticFeature,
+  InitializeParams,
+  ServerCapabilities
+} from "vscode-languageclient";
+
+export interface TreeViewProvider {}
+
+export class MetalsFeatures implements StaticFeature {
+  treeViewProvider?: TreeViewProvider;
+  fillInitializeParams?: (params: InitializeParams) => void;
```

Comment:
This method can be omitted since it's optional and it's not implemented

```suggestion
```

Reply:
I just implemented this method to add the experimental `treeViewProvider` field.